### PR TITLE
Discord.bio presence

### PR DESCRIPTION
**Pull request by MauriceX24**
image

### CODE REVIEW

File: discordbio/dist/presence.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    playback =
+        data.iframe_video.duration !== null
+            ? true : false;
+    if (playback) {
+        iFrameVideo = data.iframe_video.iFrameVideo;
+        currentTime = data.iframe_video.currTime;
+        duration = data.iframe_video.dur;
+        paused = data.iframe_video.paused;
+    }
+});
```
Comment (Bas950) [inline discordbio/dist/presence.js]: you dont have an iFrame so why is this here?

File: discordbio/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "id": "268768664411439115"
+  },
+  "service": "Discord.bio",
+  "description": {
+    "en": "Create a profile in under 30 seconds.",
+    "de": "Erstelle ein Profil in weniger als 30 sekunden."
+  },
+  "url": ["discord.bio"],
+  "version": "1.0",
+  "logo": "https://i.imgur.com/19jXt3Z.jpg",
```
Comment (frozaum) [inline discordbio/dist/metadata.json]: minimal comment (not necessary to change) but i think the logo is a bit giant xd

File: discordbio/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  "author": {
+    "name": "MauriceX",
+    "id": "268768664411439115"
+  },
+  "service": "Discord.bio",
+  "description": {
+    "en": "Create a profile in under 30 seconds.",
+    "de": "Erstelle ein Profil in weniger als 30 sekunden."
+  },
+  "url": ["discord.bio"],
```
Comment (Bas950) [inline discordbio/dist/metadata.json]: URL most be in string if its only 1

Comment (MauriceX24) [reply discordbio/dist/metadata.json]: Should be done

File: discordbio/dist/metadata.json (hunk 2 above)
Comment (MauriceX24) [reply discordbio/dist/metadata.json]: Changed. :D

File: discordbio/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  "author": {
+    "name": "MauriceX",
+    "id": "268768664411439115"
+  },
+  "service": "Discord.bio",
+  "description": {
+    "en": "Create a profile in under 30 seconds.",
+    "de": "Erstelle ein Profil in weniger als 30 sekunden."
+  },
+  "url": ['discord.bio'],
```
Comment (Bas950) [inline discordbio/dist/metadata.json]: "url": "discord.bio",

Comment (MauriceX24) [reply discordbio/dist/metadata.json]: ohh thanks - i thought the [] is important

Comment (Bas950) [reply discordbio/dist/metadata.json]: [] is for arrays
